UnitaryTesting/TestDemeritPointsCalculator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ExercisesMoshCourse;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitaryTesting
+{
+    [TestClass]
+    public class TestDemeritPointsCalculator
```

Comment:
Similar to testing class/filenames. It's the class under test + tests wording: `DemeritPointsCalculatorTests`